WooCommerce/WooCommerceTests/Authentication/Store Creation/Progress/StoreCreationProgressViewModelTests.swift

```diff
@@ -0,0 +1,166 @@
+import XCTest
+@testable import WooCommerce
+
+final class StoreCreationProgressViewModelTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
```

Comment:
nit: these might not be needed since there's no override logic
```suggestion
```

Reply:
Done in 29b435d26bd5c9c0b7a24b3a495aaf2a36bbec88